rootfs/usr/local/etc/php/php.ini

```diff
@@ -0,0 +1,63 @@
+[PHP]
+engine = On
+short_open_tag = Off
+asp_tags = Off
+precision = 14
+output_buffering = 4096
+zlib.output_compression = Off
+implicit_flush = Off
+unserialize_callback_func =
+serialize_precision = 17
+disable_functions =
+disable_classes =
+zend.enable_gc = On
+expose_php = On
+max_execution_time = 30
+max_input_time = 60
+memory_limit = 256M
+error_reporting = Off
```

Comment:
Wouldn't this hide different error levels?

Reply:
Changed to `E_ALL`